docs/api/index.md

```diff
@@ -472,6 +509,10 @@ When you use `test` or `bench` in the top level of file, they are collected as p
   describe.todo.concurrent(/* ... */) // or describe.concurrent.todo(/* ... */)
   ```
 
+::: warning
+You cannot use this syntax, when using Vitest as [type checker](/guide/testing-types).
+:::
```

Comment:
This feels a bit redundent to repeat, maybe we just mention you can't use them in the `test-types.md` instead?

Reply:
I think we try to write self contained docs, no? It feels kind of weird to me to move a warning that you can't use certain API from its API page 🤔